src/System.ServiceModel.Primitives/ref/project.json

```diff
@@ -13,6 +13,8 @@
     "System.Reflection": "4.0.0",
     "System.Net.Primitives": "4.0.0",
     "System.Security.Principal": "4.0.0",
+    "System.Security.Cryptography.X509Certificates" : "4.0.0-beta-*",
```

Comment:
I think the assembly and contract version is correct - we have to make sure the packages are being generated with the right version.